mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -1965,6 +2033,116 @@ resource "google_project_iam_member" "composer-worker" {
 `, environment, network, subnetwork, serviceAccount)
 }
 
+func testAccComposer1Environment_nodeCfg_withTags(environment, network, subnetwork, serviceAccount string) string {
```

Comment:
Please add fields from node_config compatible with Composer 1 here, which are not tested yet:
* machine_type
* disk_size_gb
* oauth_scopes